apps/theming-designer/src/components/AccessibilityDetailsList.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import {
+  DetailsList,
+  DetailsRow,
+  IDetailsRowStyles,
+  IDetailsRowProps,
+  IColumn,
+  IGroup,
+  MessageBar,
+  MessageBarType,
+  SelectionMode,
+  ITheme
+} from '../../../../packages/office-ui-fabric-react/lib/index';
```

Comment:
Again, can you import these from `office-ui-fabric-react` directly?

Reply:
Yea weird in the draft PR one of the comments was to use lib/index so I thought I had to do that but it looks like it works without that.

Reply:
You should never import things from a different package by relative path (at least in UI code--might be some weird cases in build code where that's needed). If the compiler complains about being unable to resolve the files, there's something you can add to the tsconfig and webpack config to fix it.